### PR TITLE
Updating analyses with the modulefile and also adding in machine.conf…

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
… ites

## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

This PR:

Removes all the run*.sh files because they will not be used in para or production.
Removes the module loads from the driver scripts and into modulefiles
Removes items that were fixed in the update to machine.conf.  

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Run one or all of the dev/driver/script files, specifying a cyc time in the qsub command the cyc is only needed for stats, not for plots:

qsub -v cyc=00 /EVS/dev/driver/scripts/analyses/stats/jevs_*_stats.sh 
qsub /EVS/dev/driver/scripts/analyses/plots/jevs_analyses_plots.sh

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

ASAP.  

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
@PerryShafran-NOAA -- should I comment out the following lines in your driver scripts? Or can they remain active?

`export MET_bin_exec=bin`
`export metplus_verbosity=DEBUG`
`export met_verbosity=2`
`export log_met_output_to_metplus=yes`

**Comment by PerryShafran-NOAA** [comment]
Commenting them out is fine.  They are not needed either way, since machine.conf is called when METplus is run.

Perry

**Comment by ShelleyMelchior-NOAA** [comment]
@PerryShafran-NOAA  -- I ran the analyses stats processing. Please review.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/analyses`
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_rtma_ru_stats_00.o81815643`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_rtma_stats_00.o81815657`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_urma_stats_00.o81815682`

**Comment by PerryShafran-NOAA** [comment]
Everything looks good in terms of these stat files.

In order to run the plot routine, you'll need to bring in 31 days worth of stats from the parallel.  Let me know if they are there, I'm not sure I transferred this particular directory of stats files from cactus.

Perry

**Comment by ShelleyMelchior-NOAA** [comment]
@PerryShafran-NOAA -- I ran the analyses plots processing. Please review.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/analyses`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_rtma_plots_00.o81878797`

**Comment by PerryShafran-NOAA** [comment]
The analyses plots file is in good shape.  

Perry

**Comment by ShelleyMelchior-NOAA** [comment]
Great! Doing a quick code review next.

File: parm/evs_config/analyses/config.evs.rtma.prod [hunk 1]
```diff
@@ -44,9 +44,9 @@ export end_date=$yday
 #metplus_verbosity:         DEBUG, INFO, WARN, ERROR (most verbose is DEBUG)
 #met_verbosity:             MET verbosity for logging, 1-5 (most verbose is 5)
 #log_met_output_to_metplus: log MET output to METplus log, yes or no
-export metplus_verbosity="DEBUG"
-export met_verbosity="2"
-export log_met_output_to_metplus="yes"
+#export metplus_verbosity="DEBUG"
+#export met_verbosity="2"
+#export log_met_output_to_metplus="yes"
 ## DATA DIRECTIVE SETTINGS
```
Comment (ShelleyMelchior-NOAA) [inline parm/evs_config/analyses/config.evs.rtma.prod:43]: Do you want lines 47-49 formally removed, or keep them in as commented out? Same question for the equivalent parm file for urma.

Comment (PerryShafran-NOAA) [reply parm/evs_config/analyses/config.evs.rtma.prod:43]: Removed.  Let me fix that.

Comment (PerryShafran-NOAA) [reply parm/evs_config/analyses/config.evs.rtma.prod:43]: Fixed.

**Comment by PerryShafran-NOAA** [comment]
Hold this PR for a second.  One of the analyses plots is missing and I need to figure out why.  I might need to figure that out.  If it's not an easy fix, approve this PR and I'll submit a new PR for the fix to this plot.

Perry

**Comment by ShelleyMelchior-NOAA** [comment]
Can you change all of these lines in your driver scripts
`source $HOMEevs/modulefiles/analyses/analyses_stats.sh`
to
`source $HOMEevs/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh`

**Comment by PerryShafran-NOAA** [comment]
Yes I can.  

Perry

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
All code changes are as expected. This PR is ready for merge.

**Pull request merged**